packages/translate/src/index.js

```diff
@@ -183,17 +185,60 @@ Translate.prototype.detect = function(input, callback) {
  *     // ]
  *   }
  * });
+ *
+ * //-
+ * // Get the names of the languages in a target language:
```

Comment:
See the languages to which you can translate text of a specific language.

^ If that reads weird, feel free to translate that to better English.